rocketpy/motors/motor.py

```diff
@@ -1179,6 +1214,145 @@ def __init__(
         self.prints = _MotorPrints(self)
         self.plots = _MotorPlots(self)
 
+    @staticmethod
+    def load_from_eng_file(
+        file_name,
+        nozzle_radius,
```

Comment:
My suggestion:

Don't make `nozzle_radius` a mandatory argument. If user doesn`t provide it, set the noozle radius as a percentage of the chamber radius. Let this clear on the documentation.


```suggestion
        nozzle_radius = None,
```

Reply:
ok